shell/platform/tizen/public/flutter_tizen.h

```diff
@@ -183,6 +203,21 @@ FLUTTER_EXPORT void FlutterDesktopViewResize(FlutterDesktopViewRef view,
                                              int32_t width,
                                              int32_t height);
 
+FLUTTER_EXPORT void FlutterDesktopViewOnPointerEvent(
+    FlutterDesktopViewRef view,
+    FlutterDesktopPointerEventType type,
+    double x,
+    double y,
+    size_t timestamp,
+    int32_t device_id);
```

Comment:
Doesn't NUI support scroll events?

Reply:
I haven't checked dali's scroll event yet.
I want to work on this in another PR if you are ok.